modules/cert-manager-override-flag-controller.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-flag-controller_{context}"]
+= Customizing the cert-manager controller by overriding flags
```

Comment:
```suggestion
= Customizing cert-manager to automatically delete TLS secret when certificate resource is removed
```

This section seems bit repetitive as it also is part of spec.controllerConfig.overrideArgs which was in previous section as well. Since this is an important feature with a warning, IMHO should be fine to add a separate section but renaming it accordingly to reflect the use case of enable-certificate-owner-ref.

Reply:
Can we call it "Managing TLS secrets and certificates"

Reply:
The heading will be long otherwise. We can explain that cert-manager deletes the secret automatically  when certificate resource is removed in the paragraph.